.readthedocs.yml

```
@@ -24,7 +24,7 @@ build:
       # If there are no changes (git diff exits with 0) we force the command to return with 183.
       # This is a special exit code on Read the Docs that will cancel the build immediately.
       - |
-        if [ "$READTHEDOCS_VERSION_TYPE" = "external" ] && git diff --quiet origin/main -- docs/;
+        if [ "$READTHEDOCS_VERSION_TYPE" = "external" ] && ! ( git diff --quiet origin/main -- docs/ || git diff --quiet origin/main -- requirements/docs.txt || git diff --quiet origin/main -- .readthedocs.yml );
```

```suggestion
        if [ "$READTHEDOCS_VERSION_TYPE" = "external" ] && git diff --quiet origin/main -- docs/ .readthedocs.yml requirements/docs.txt;
```

I think we can just add each of these files in 1 command.